foqus_lib/conftest.py

```diff
@@ -49,22 +56,56 @@ def psuade_path():
     return Path(_psuade_path).resolve()
 
 
-@pytest.fixture(scope="module", params=["UQ/Rosenbrock.foqus"])
-def flowsheet_session_file(examples_dir, request):
-    return str(examples_dir / "test_files" / request.param)
+@pytest.fixture(
+    scope="session",
+)
+def foqus_working_dir(
+    request, tmp_path_factory, name: str = "foqus-working-dir"
+) -> Path:
+    d = tmp_path_factory.mktemp(name, numbered=False)
+
+    what = "The FOQUS working directory for this test run"
```

Comment:
Yeah, I agree that the naming for this is not the clearest (and I admit this would be far from being the worst offender within the `conftest.py`/`pytest_qt_extras.py` code...). Actually, I've since added better reporting functionality for displaying the working directory (that, unlike `print()` or `logging`, is also shown when pytest is run without the `-s` flag), so I think we can get rid of those printouts entirely.